.github/workflows/main.yml

```
@@ -55,30 +53,23 @@ jobs:
       run: flake8 . --count --exit-zero --max-complexity=10 --statistics
 
     - name: Run unit and integration tests
-      run: coverage run --rcfile=setup.cfg -m unittest discover
+      run: coverage run --rcfile=pyproject.toml -m unittest discover
 
     - name: Publish results to coveralls upon success
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         COVERALLS_PARALLEL: true
         COVERALLS_FLAG_NAME: run-${{ matrix.python-version }}-${{ matrix.os }}-${{ matrix.install-extras }}
-      run: coveralls --rcfile=setup.cfg --service=github || true
+      run: coveralls --rcfile=pyproject.toml --service=github || true
```

```suggestion
      run: coveralls --rcfile=pyproject.toml --service=github
```
I think this might be the issue. It looks like things started breaking when this option was removed from the finish job below. 

From coveralls and the error message in GA, it looks like the finish statement is looking for a build number one more than what's been run (8470998746 vs 8470998745). Not sure if this will get the build numbers back in sync.